public/tabs/chat/chat_page_content.tsx

```diff
@@ -44,12 +53,29 @@ export const ChatPageContent: React.FC<ChatPageContentProps> = React.memo((props
   const chatActions = useChatActions();
   const registry = getIncontextInsightRegistry();
   const configSchema = getConfigSchema();
+  const latestInputRef = useRef<HTMLDivElement>(null);
+  const [messageSpacerHeight, setMessageSpacerHeight] = useState(0);
 
   useLayoutEffect(() => {
-    if (!props.chatScrollTopRef.current) {
+    if (loading && latestInputRef.current) {
+      setMessageSpacerHeight(
+        window.innerHeight - latestInputRef.current.clientHeight - CHAT_PAGE_CONTENT_PADDING_HEIGHT
```

Comment:
Do we need to consider `latestInputRef.current.clientHeight` greater than `window.innerHeight` here?

Reply:
If the height value becomes negative, then setting it to minHeight would be equivalent to setting minHeight to 0, resulting in no spacing being provided, which is expected. But do you think it is necessary to add a conditional check here?

Reply:
I'm not sure if we need to use `latestInputRef.current.clientHeight - window.innerHeight` if input height greater than `window.innerHeight`. Because I think the current response should still at the top of chat page content. I'm OK to leave it as current implementation. Let's fix this in the future PR.